tests/ui_tools/test_popups.py

```diff
@@ -550,24 +550,62 @@ def test_keypress_navigation(self, mocker, widget_size,
         super_keypress.assert_called_once_with(size, expected_key)
 
 
+@pytest.mark.parametrize('stream, stream_id, server_feature_level', [
```

Comment:
Yes. I thought it will increase our scope of testing, and potentially detect/eliminate any bug that may get unnoticed, if we tried to localize the parameters inside the class?

Reply:
I have removed the class parameterization and moved the fixtures to another test function(`test_popup_height`), since the popup height will change depending on the server version running - which corresponds to the numbers of fields supported in that version. :)